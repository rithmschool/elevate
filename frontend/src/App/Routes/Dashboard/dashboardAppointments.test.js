import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import DashboardAppointments from "./dashboardAppointments";

it("renders without crashing", function() {
  shallow(<DashboardAppointments />);
});

it("matches snapshot", function() {
  let wrapper = shallow(<DashboardAppointments />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
