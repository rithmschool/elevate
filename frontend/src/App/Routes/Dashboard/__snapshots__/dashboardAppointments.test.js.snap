// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h1>
    Appointments
  </h1>
</div>
`;
